ft_calibration_node:
  ros__parameters:
    topic_raw_wrench: /follower/raw_wrench
    topic_joint_state: /follower/joint_states
    reference_frame:
      id: base_link
    sensor_frame:
      id: ft_sensor
    gravity_in_reference_frame:
      - 0.0
      - 0.0
      - -9.81
    calibration:
      min_nb_samples: 10
      filename: "calib.yaml"
    kinematics:
      plugin_name: kinematics_interface_kdl/KinematicsInterfaceKDL
      plugin_package: kinematics_interface
      base: base_link
      tip: tool0
      group_name: arm
      alpha: 0.0005
